assets/src/containers/DashboardAppBar.js

```diff
@@ -71,7 +72,7 @@ function DashboardAppBar (props) {
               </Link>
               <Link
                 to={{ pathname: `/courses/${courseId}` }}
-                className={classes.homeButton}>
+                className={clsx(classes.homeButton, user.user_defaults.demomode === "True" ? 'sensitive' : '')}>
```

Comment:
I thought I'd be using clsx more places. This was used in the material ui documentation and seemed pretty useful.